docs/data/material/getting-started/templates/dashboard/components/SideMenu.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import MuiDrawer from '@mui/material/Drawer';
+import List from '@mui/material/List';
+import Box from '@mui/material/Box';
+import Divider from '@mui/material/Divider';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import HomeOutlinedIcon from '@mui/icons-material/HomeOutlined';
+import AnalyticsOutlinedIcon from '@mui/icons-material/AnalyticsOutlined';
+import PeopleOutlinedIcon from '@mui/icons-material/PeopleOutlined';
+import AssignmentOutlinedIcon from '@mui/icons-material/AssignmentOutlined';
+import SettingsOutlinedIcon from '@mui/icons-material/SettingsOutlined';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import HelpOutlineOutlinedIcon from '@mui/icons-material/HelpOutlineOutlined';
+
+const drawerWidth = 180;
+
+const Drawer = styled(MuiDrawer)({
+  width: drawerWidth,
+  flexShrink: 0,
+  whiteSpace: 'nowrap',
+  boxSizing: 'border-box',
+  '& .MuiDrawer-paper': {
+    width: drawerWidth,
+    boxSizing: 'border-box',
+  },
+});
+
+export default function SideMenu() {
+  return (
+    <Drawer
+      variant="permanent"
+      sx={(theme) => ({
+        display: { xs: 'none', md: 'block' },
+        '.MuiPaper-root': {
+          backgroundColor: theme.palette.background.paper,
+        },
+      })}
+    >
+      <Box sx={{ height: 64 }} />
+      <Divider />
+      <List>
+        {[
+          { text: 'Home', icon: <HomeOutlinedIcon /> },
+          { text: 'Analytics', icon: <AnalyticsOutlinedIcon /> },
+          { text: 'Clients', icon: <PeopleOutlinedIcon /> },
+          { text: 'Tasks', icon: <AssignmentOutlinedIcon /> },
+        ].map((item, index) => (
+          <ListItem key={index} disablePadding sx={{ display: 'block' }}>
+            <ListItemButton
```

Comment:
I think we should make them full width and increase the padding 🤔 
![image](https://github.com/mui/material-ui/assets/72460825/8134227a-fa78-435b-b617-7add205322df)